arcgis-ios-sdk-samples/Features/Display feature layers/README.metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "category": "Features",
+    "description": "Display feature layers from various data sources.",
+    "ignore": false,
+    "images": [
+        "display-feature-layers.png"
+    ],
+    "keywords": [
+        "feature",
+        "geodatabase",
+        "geopackage",
+        "layers",
+        "service",
+        "shapefile",
+        "table",
+        "AGSFeatureLayer",
+        "AGSGeoPackageFeatureTable",
+        "AGSGeodatabase",
+        "AGSPortalItem",
+        "AGSServiceFeatureTable",
+        "AGSShapefileFeatureTable"
+    ],
+    "redirect_from": [],
```

Comment:
This will need to include links to all the old feature layer samples being removed in favor of this new sample